From 00d1e380fc18afc08b1901f12df1ba1fb79ddfee Mon Sep 17 00:00:00 2001
From: Noam Lessner <nlessner@habana.ai>
Date: Mon, 17 Feb 2020 10:06:30 +0200
Subject: [PATCH] [SW-8229] goya2: all targets flash offset is 4MB

Align all target to offset of 4MB in flash so we dont need
special images for palladium. Because palladium emulates only
a 8MB flash, unlike asic and fpga which have the full 16MB flash.

Change-Id: I81c3854faa044f8edbf22b36669569f1bc657627
Signed-off-by: Noam Lessner <nlessner@habana.ai>
---

diff --git a/Makefile b/Makefile
index abd4c21..e9c8c05 100644
--- a/Makefile
+++ b/Makefile
@@ -25,11 +25,7 @@
 ARCBOOT_CFLAGS += -I$(dir $(BR2_EXTERNAL))/embedded/specs
 ARCBOOT_CFLAGS += -DBTLVER=$(ARCBOOT_VER)
 ARCBOOT_CFLAGS += -DBTLID=0x30
-ifneq ($(HL_TARGET),pldm)
-	ARCBOOT_CFLAGS += -DFLASH_START_OFFSET=0xC00000
-else
-	ARCBOOT_CFLAGS += -DFLASH_START_OFFSET=0x0
-endif
+ARCBOOT_CFLAGS += -DFLASH_START_OFFSET=0x400000
 
 LDFLAGS = -flto
 LDFLAGS += -L $(LIBDIR) -lsb -lellipsys -L $(HL_DEV)
